src/relax/transform/fuse_ops.cc

```diff
@@ -395,14 +394,8 @@ class FunctionCreator : public ExprMutator {
             }
           }
 
-          for (const Expr& arg : call->args) {
-            CheckDefAndUpdateParam(arg);
-          }
+          CheckDefAndUpdateParams(call, call->args);
         }
-      } else {
-        const auto* tuple_item = var_binding->value.as<TupleGetItemNode>();
-        ICHECK(tuple_item != nullptr);
-        CheckDefAndUpdateParam(tuple_item->tuple);
```

Comment:
cc @Hzfengsy, I'm not sure what this code path is for. I can pass all tests without this.

Reply:
It is used for the TupleGetItem node. I'm surprised that it is not covered by the tests